wp-content/themes/twentytwenty-child/functions.php

```diff
@@ -293,4 +338,232 @@ function update_students_settings() {
+    echo '<input type="checkbox" id="show_address" name="address_box_ajax" value="1" ' . $checked . '  />';
+}
+
+/**
+ *  Callback for a custom setting 
+ * */
+function show_checkbox_birthdate_ajax() {
+    $options = get_option( 'show_ajax_settings' );
+    $checked = '';
+    if( $options["show_birthdate"] == 1 ) $checked = 'checked';
+    echo '<input type="checkbox" id="show_birthdate" name="birthdate_box_ajax" value="1" ' . $checked . '  />';
+}
+
+/**
+ *  Callback for a custom setting 
+ * */
+function show_checkbox_class_ajax() {
+    $options = get_option( 'show_ajax_settings' );
+    $checked = '';
+    if( $options["show_class"] == 1 ) $checked = 'checked';
+    echo '<input type="checkbox" id="show_class" name="class_box_ajax" value="1" ' . $checked . '  />';
+}
+
+/** 
+ * Callback for a custom setting 
+ * */
+function show_checkbox_status_ajax() {
+    $options = get_option( 'show_ajax_settings' );
+    $checked = '';
+    if( $options["show_status"] == 1 ) $checked = 'checked';
+    echo '<input type="checkbox" id="show_status" name="status_box_ajax" value="1" ' . $checked . '  /> </div> </form>';
+}
+
+/**
+ *  Function executed by AJAX when a checkbox is un/checked on the settings page 
+ * */
+function ajax_func() {
+    $options = get_option( 'show_ajax_settings' );
+    $options[ $_POST['option'] ] = 0;
+    if( $_POST['checked'] === 'true' ) {
+        $options[$_POST['option']] = 1;
+    }
+    update_option( 'show_ajax_settings', $options );
+}
+add_action( 'wp_ajax_nopriv_ajax_func', 'ajax_func' );
+add_action( 'wp_ajax_ajax_func', 'ajax_func' );
+
+/** 
+ * Add a column for the activity status on the student dashboard 
+ * */
+function add_student_columns( $columns ) {
+    return array_merge( $columns, array( 'Active' => __( 'Active' ) ) );
+}
+add_filter('manage_student_posts_columns' , 'add_student_columns');
+
+/**
+ * Print the activity checkboxes on the dashboard
+ *  */ 
+function print_extra_columns() { 
+    $meta = get_post_meta( get_the_ID() );
+    $checked = '';
+    if( $meta["status"][0] == 1 ) $checked = 'checked';
+    echo '<form method="post"> <div > <input type="checkbox" class="student-status" id="' . get_the_ID() . '" name="status" value="1" ' . $checked . '  /> </div> </form>';
+}
+add_action( 'manage_student_posts_custom_column', 'print_extra_columns' );
+
+/**
+ * Make the custom column sortable
+ */
+function sort_student_status( $columns ) {
+    $columns['Active'] = __('status');
+    return $columns;
+}
+add_filter( 'manage_edit-student_sortable_columns', 'sort_student_status' );
+
+/**
+ * Define how to order our sortable custom column
+ */
+function student_status_orderby( $query ) {
+    if( ! is_admin() ) return;
+ 
+    $orderby = $query->get( 'orderby' );
+ 
+    if( 'status' == $orderby ) {
+        $query->set('meta_key','status');
+        $query->set('orderby','meta_value');
+    }
+}
+add_action( 'pre_get_posts', 'student_status_orderby' );
```

Comment:
Here you just checking if it's on the admin side and not for the post type. This means this code will apply for all post type. Let's fix that.